gammapy/irf/psf_analytical.py

```diff
@@ -354,3 +358,46 @@ def info(self, fractions=[0.68, 0.95], energies=Quantity([1., 10.], 'TeV'),
                            "E = {2:4.1f}: {3:5.8f}\n"
                            "".format(100 * fraction, theta, energy, radius))
         return ss
+
+
+
+    def to_table_psf(self, theta=None, offset_max=None, binsz=0.01, exposure=None):
+        """
+        Convert triple Gaussian PSF ot table PSF.
+
+        Parameters
+        ----------
+        theta : `~astropy.coordinates.Angle`
+            Observation offset. Default theta = 0 deg.
+        offset_max : `~astropy.coordinates.Angle`
+            Maximum offset from PSF center. Default offset_max = 2.5 deg
+        binsz : float
+            Size of the spatial bins in deg / pixel. Default binsz = 0.01 deg
+        exposure : `~astropy.units.Quantity`
+            Energy dependent exposure. Should be in units equivalent to 'cm^2 s'.
+
+        Returns
+        -------
+        tabe_psf : `~gammapy.irf.EnergyDependentTablePSF`
+            Instance of `EnergyDependentTablePSF`.
+        """
+        # Convert energies to log center
+        ebounds = EnergyBounds.from_lower_and_upper_bounds(self.energy_lo, self.energy_hi)
+        energies = ebounds.log_centers
+        
+        # Default for exposure
+        exposure = exposure or Quantity(np.ones(len(energies)), 'cm^2 s')
```

Comment:
It would probably be better to move this default for `exposure` to the `EnergyDependentTablePSF` class.
It's in EnergyDependentTablePSF because it's there for Fermi.
But for IACTs we probably won't always have it / use it.

So since this weighting is something that's done in EnergyDependentTablePSF, not here, I'd prefer to also have the default there, and then here you can just take `exposure=None` in and pass it along to the EnergyDependentTablePSF initialiser in case someone does want to set it.